components/layout_2020/Cargo.toml

```diff
@@ -25,6 +25,7 @@ ipc-channel = "0.12"
 libc = "0.2"
 msg = {path = "../msg"}
 net_traits = {path = "../net_traits"}
+once_cell = "1.2.0"
```

Comment:
We already use `mitochondria`, which has an `OnceCell`, I think.

Reply:
Switched